src/unlimited.cpp

```diff
@@ -494,7 +495,7 @@ UniValue expedited(const UniValue& params, bool fHelp)
     std::vector<CNode*>::iterator elem = std::find(xpeditedBlkUp.begin(), xpeditedBlkUp.end(),node); 
     if ((flags & EXPEDITED_BLOCKS)&&(flags & EXPEDITED_STOP))
       {
-	if (elem == xpeditedBlkUp.end()) xpeditedBlkUp.erase(elem);
+	if (elem != xpeditedBlkUp.end()) xpeditedBlkUp.erase(elem);
```

Comment:
It looks like the old code is wrong. According to [reference here](http://en.cppreference.com/w/cpp/container/vector/erase), `elem` must be a valid deferable iterator. Erasing an invalid iterator is UB.